servicetalk-grpc-internal/gradle/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are they needed if empty?
Consider updating copyright years.

Reply:
like the `log4j.xml` file I prefer to keep the stubs there for consistency reasons.